tests/experimental/metric_repository/test_column_descriptive_metrics_metric_retriever.py

```diff
@@ -142,3 +150,135 @@ def test_get_metrics():
             column="col2",
         ),
     ]
+
+
+def test_get_metrics_metrics_missing():
+    """This test is meant to simulate a single metric missing from the computed metrics."""
+    mock_context = Mock(spec=CloudDataContext)
+    mock_validator = Mock(spec=Validator)
+    mock_context.get_validator.return_value = mock_validator
+    mock_validator.compute_metrics.return_value = {
+        # ("table.row_count", (), ()): 2, # Missing table.row_count metric
+        ("table.columns", (), ()): ["col1", "col2"],
+        ("table.column_types", (), "include_nested=True"): [
+            {"name": "col1", "type": "float"},
+            {"name": "col2", "type": "float"},
+        ],
+        # ("column.min", "column=col1", ()): 2.5, # Missing column.min metric for col1
+        ("column.min", "column=col2", ()): 2.7,
+        ("column.max", "column=col1", ()): 5.5,
+        ("column.max", "column=col2", ()): 5.7,
+        ("column.mean", "column=col1", ()): 2.5,
+        ("column.mean", "column=col2", ()): 2.7,
+        ("column.median", "column=col1", ()): 2.5,
+        ("column.median", "column=col2", ()): 2.7,
+        ("column_values.null.count", "column=col1", ()): 1,
+        ("column_values.null.count", "column=col2", ()): 1,
+    }
+    mock_batch = Mock(spec=Batch)
+    mock_batch.id = "batch_id"
+    mock_validator.active_batch = mock_batch
+
+    metric_retriever = ColumnDescriptiveMetricsMetricRetriever(context=mock_context)
+
+    mock_batch_request = Mock(spec=BatchRequest)
+
+    with mock.patch(
+        f"{ColumnDomainBuilder.__module__}.{ColumnDomainBuilder.__name__}.get_effective_column_names",
+        return_value=["col1", "col2"],
+    ):
+        metrics = metric_retriever.get_metrics(batch_request=mock_batch_request)
+
+    assert metrics == [
+        TableMetric[int](
+            batch_id="batch_id",
+            metric_name="table.row_count",
+            value=None,
+            exception=MetricException(type="TBD", message="TBD"),
```

Comment:
Here is an example of the placeholder exception flowing through to the metric if it is missing. Note also the value is None.